Packages/com.unity.inputsystem/InputSystem/Actions/InputActionState.cs

```diff
@@ -1435,20 +1443,13 @@ private bool IsConflictingInput(ref TriggerState trigger, int actionIndex)
 
             // If we're not really effecting any change on the action, ignore the control state change.
             // NOTE: We may be looking at a control here that points in a completely direction, for example, even
-            //       though it has the same magnitude. However, we require a control to *higher* absolute actuation
+            //       though it has the same magnitude. However, we require a control to *increase* absolute actuation
             //       before we let it drive the action.
-            if (Mathf.Approximately(trigger.magnitude, actionState->magnitude))
+            if (!isControlCurrentlyDrivingTheAction && Mathf.Approximately(trigger.magnitude, actionState->magnitude))
```

Comment:
Sidenote: if you press d-pad *left* on one gamepad and then press d-pad *right* on a second gamepad, the second input will be *ignored*. The current code makes it a requirement that the level of actuation is higher. So the d-pad *right* press will only come through once you release the d-pad *left* input.